e2e-tests/playwright/e2e/ocm.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test } from '@playwright/test';
+import { Common } from '../utils/Common';
+import { UIhelper } from '../utils/UIhelper';
+import { Clusters } from '../support/pages/Clusters';
+
+//Pre-req: Enable janus-idp-backstage-plugin-ocm-backend-dynamic and janus-idp-backstage-plugin-ocm Plugins
+
+const clusterDetails = {
+  clusterName: 'testCluster',
+  status: 'Ready',
+  platform: 'IBM',
+  cpuCores: '12',
+  memorySize: '47 Gi',
+  ocVersion: '4.13.23',
+};
+
+test.describe.skip('Test OCM plugin', () => {
+  test.beforeEach(async ({ page }) => {
```

Comment:
Same comment that @subhashkhileri Subashi added on the other review comment.

Can we use the beforeAll hook instead of beforeEach to avoid logging in for every test?

Reply:
@josephca This entire test it's being skipped, I will create a new Jira to enable this one